spec/code_coverage_helper.rb

```diff
@@ -6,5 +6,6 @@
   SimpleCov.start("rails") do
     add_filter "spec"
     add_filter "vendor"
+    minimum_coverage((ENV["CODE_COVERAGE_PERCENTAGE"] || 0).to_i)
```

Comment:
Does using `ENV.fetch` here improve things?

(I really don't know.)

Reply:
I don't think it matters.
I believe fetch behaves the same as Hash's fetch, even though ENV.class == 'Object' and ENV.method(:fetch).source_location == nil, which is quite odd.

Reply:
I suppose
`minimum_coverage(ENV.fetch("CODE_COVERAGE_PERCENTAGE", 0).to_i)`
Look slightly cleaner 🤔